composer.json

```diff
@@ -25,7 +25,7 @@
         }
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
         "laminas/laminas-stdlib": "^3.2.1"
```

Comment:
Want me to increase this to something not ancient as well? 3.2.1 is not even compatible with PHP 8 :grin: